diff --git a/internal/registry/devices.go b/internal/registry/devices.go
--- a/internal/registry/devices.go
+++ b/internal/registry/devices.go
@@ -3,8 +3,9 @@
 import (
 	_ "github.com/Alia5/VIIPER/device/dualsense"
 	_ "github.com/Alia5/VIIPER/device/dualshock4"
 	_ "github.com/Alia5/VIIPER/device/keyboard"
 	_ "github.com/Alia5/VIIPER/device/mouse"
 	_ "github.com/Alia5/VIIPER/device/ns2pro"
+	_ "github.com/Alia5/VIIPER/device/nspro"
 	_ "github.com/Alia5/VIIPER/device/xbox360"
 )
